README.md

```diff
@@ -412,6 +412,10 @@ Samples are in the [`samples/`](https://github.com/googleapis/java-storage/tree/
 
 To get help, follow the instructions in the [shared Troubleshooting document][troubleshooting].
 
+## Transport
+
+Cloud Storage uses gRPC for the transport layer.
```

Comment:
@sydney-munro we probably don't want this change as @BenWhitehead explained that this trasnport is still in beta. We have https://github.com/googleapis/sdk-platform-java/pull/3052 to address this